src/main/java/org/dataconservancy/pass/keyword/service/PassKeywordService.java

```diff
@@ -0,0 +1,69 @@
+package org.dataconservancy.pass.keyword.service;
+
+import cc.mallet.util.*;
+import cc.mallet.types.*;
+import cc.mallet.pipe.*;
+import cc.mallet.pipe.iterator.*;
+import cc.mallet.topics.*;
+
+import java.util.*;
+import java.util.regex.*;
+import java.io.*;
+
+import java.io.IOException;
+import java.lang.Exception;
+
+public class PassKeywordService {
+
+    public String trainParallelTopicModel(int numTopics, InstanceList instances) throws IOException {
+        // Create a model with 100 topics, alpha_t = 0.01, beta_w = 0.01
+        //  Note that the first parameter is passed as the sum over topics, while
+        //  the second is the parameter for a single dimension of the Dirichlet prior.
+        ParallelTopicModel model = new ParallelTopicModel(numTopics, 1.0, 0.01);
+
+        model.addInstances(instances);
+
+        // Use two parallel samplers, which each look at one half the corpus and combine
+        //  statistics after every iteration.
+        model.setNumThreads(2);
+
+        int iterations = 500;
+        // Run the model for iterations and stop (50 is for testing only,
+        //  for real applications, use 1000 to 2000 iterations)
+        model.setNumIterations(iterations);
+        model.estimate();
+
+        // Save model after iterations
+        String modelPath = "\\\\wsl.localhost\\Ubuntu-18.04\\home\\jkim25\\PASS\\pass-keyword-service\\src\\main\\java\\org\\dataconservancy\\pass\\keyword\\service\\model.dat";
```

Comment:
Instead load the model from the classpath. Look at this.getClass().getResourceAsStream()